src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -70,7 +70,7 @@ export class VmWizardComponent implements WizardConfiguration {
   private isolatedGpuPciIds: string[];
 
   entityWizard: EntityWizardComponent;
-  private productType = window.localStorage.getItem('product_type') as ProductType;
+  private productType = this.systemGeneralService.getProductType();
 
   wizardConfig: Wizard[] = [
```

Comment:
Is it possible to move **wizardConfig: Wizard[] = []** data<array> outside of this component? (so it will have less lines)

Reply:
Reordered some fields, but generally refactoring vm wizard is outside of scope of this PR.

Reply:
okay